src/cmds/scripts/pbs_habitat.in

```diff
@@ -656,6 +665,7 @@ if [ "${PBS_START_SERVER:-0}" != 0 ] ; then
 		else
 			#Remove old dataservice directory
 			if [ `echo "${old_inst_dir}"| grep "pgsql.old"` ]; then
+				backupdir "$old_inst_dir" "$PBS_HOME"
```

Comment:
We might have to ensure that backup activity was successful before deleting old_inst_dir.

Reply:
Hey, thanks for the reviewing I have now added a message on backup fail.